resources/guis.yml

```diff
@@ -334,3 +334,13 @@ guis:
   price: Free trial, then $37
   license: Proprietary
   order: 37
+- name: GitUI
+  url: https://extrawurst.itch.io/gitui
+  image_tag: guis/gitui@2x.png
+  platforms:
+  - Windows
+  - Mac
+  - Linux
+  price: Name your own price
```

Comment:
I think this may be confusing no? Maybe just say it's `free` here, and in the website just keep asking for donations

<img width="633" alt="imagem" src="https://user-images.githubusercontent.com/1999050/87217495-a9f64100-c341-11ea-8bd6-a591e386f36f.png">